ci/github/index.html

```diff
@@ -139,8 +164,6 @@ <h3 id="New_patchset">New patchset</h3>
     upstream and lead to conflicts. Commit messages should have a format like
     this:</p>
     <pre>Implement a thing
-    
-Optional extended description that should be complete sentences.
```

Comment:
I don't think this should be removed. It's intended to show the format of a commit message.

Reply:
I didn't mean to remove that